UseElastix.cmake.in

```diff
@@ -14,13 +14,8 @@ include_directories( ${ELASTIX_INCLUDE_DIRS} )
 link_directories( ${ELASTIX_LIBRARY_DIRS} )
 
 # If Elastix_FOUND is set, this file is included via find_package() which provides
-# ELASTIX_CONFIG_TARGETS_FILE and elxLIBRARY_DEPENDS_FILE. Guarding the following
-# include statements allow users to include this file directly for backwards 
-# compatibility.
+# ELASTIX_CONFIG_TARGETS_FILE.
 if( Elastix_FOUND )
```

Comment:
@kaspermarstal Should the _if_ be removed, now that I removed the entire _else_ clause?

Reply:
Yes!

Reply:
Marius wondered if there shouldn't be an else-clause, producing an error message when Elastix is not found. But you think that's not necessary, right?